site/docs/v1/tech/apis/_apikey-put-request-body.adoc

```diff
@@ -0,0 +1,14 @@
+==== Request Body
+
+[.api]
+[field]#apiKey.metaData.attributes.description# [type]#[String]# [optional]#Optional#::
+Description of the key.
+
+[field]#apiKey.permissions.endpoints# [type]#[Object]# [optional]#Optional#::
```

Comment:
You need to add the `tenantId` here if the API key is tenant scoped. See https://github.com/FusionAuth/fusionauth-site/pull/593/files#r623463793